### PR TITLE
Update README for financialcircuit

### PR DESCRIPTION
**Pull request by fosgate29**
# Description

It had a link to a smart contract deployed on Rinkeby. Since it doesn't exist anymore, the same smart contract (Verifier.sol) was deployed on Mumbai Polygon for tests.

## Related Issue

https://github.com/eea-oasis/baseline/issues/661

## Motivation and Context

Rinkeby testnet is deprecated

## How Has This Been Tested

- deployed smart contract on Mumbai Polygon testnet
- copied smart contract code to Remix, connect it using Metamask and sent same parameters of the example and it returned "true" and tested with an invalid parameter and it returned "false".

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [ ] My code follows the code style of this project.
- [ x ] My change requires a change to the documentation.
- [ x ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: examples/financialcircuit/README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Counter parties don't know each other quotes.
 
 ## Test
 Verifier smart contract was deployed on Rinkeby for tests purpose. Here is the Etherscan link:
```
Comment (ognjenkurtic) [inline examples/financialcircuit/README.md:90]: Verifier smart contract is deployed to Polygon Mumbai for testing purposes. Here is the Polygonscan link:

File: examples/financialcircuit/README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Counter parties don't know each other quotes.
...
-https://rinkeby.etherscan.io/address/0xad6ba91bd376050e807b67cd2ff83a638b3095c2#code
+https://mumbai.polygonscan.com/address/0x14e1d3c53a42b64dde5e78b069bbd295270117b9#code
 
 You can use remix to test it since Etherscan form to Write parameters didin't work with 2 dymension array.
```
Comment (ognjenkurtic) [inline examples/financialcircuit/README.md:93]: Is this comment still valid? If yes, please fix typos in didn't and dymension. If no, please remove.

Comment (fosgate29) [reply examples/financialcircuit/README.md]: Comment is still valid. It isn't possible to use READ function from Polygonscan. I tested it using Remix.

File: examples/financialcircuit/README.md (hunk 1 above)
Comment (fosgate29) [reply examples/financialcircuit/README.md]: Thanks. Fixed.

File: examples/financialcircuit/README.md (hunk 2 above)
Comment (fosgate29) [reply examples/financialcircuit/README.md]: Typos were fixed.

**Review by ognjenkurtic — APPROVED** [review]

**Review by skosito — APPROVED** [review]

**Pull request merged**